src/js/components/Footer/stories/Simple.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Footer, grommet, Grommet, Main, Text } from 'grommet';
+
+const Simple = () => (
+  <Grommet theme={grommet}>
+    <Main
+      background="light-1"
+      elevation="large"
+      pad={{ top: 'large' }}
+      gap="large"
+    >
+      <Text margin="small" size="xsmall">
+        Main Content
+      </Text>
+      <Box flex />
+      <Footer background="light-4" justify="center">
```

Comment:
Did you try not setting `justify`?

Reply:
Footer has a default of `justify="between"`, which is better for links/sitemap content.
In this case, we need  `justify="center"` because that otherwise the content will align to the right. 
<img width="374" alt="Screen Shot 2019-12-05 at 10 37 31 AM" src="https://user-images.githubusercontent.com/6320236/70263553-7ddf6580-174b-11ea-9913-0c52d82a904d.png">